applications/maplab-server-node/include/maplab-server-node/maplab-server-node.h

```diff
@@ -47,8 +49,10 @@ struct SubmapProcess {
 };
 
 class MaplabServerNode final {
+  friend MaplabServerNodeTest;  // Test.
```

Comment:
why do we need this to be a `friend`?

Reply:
because otherwise we cannot access `total_num_merged_submaps_` in the test

Reply:
if it is only this we could simply make a getter method for this member, like `getTotalNumOfMergedSubmaps`. Sounds a bit cleaner to me than making it a `friend`. WDYT?